pkg/scaffold/util/util.go

```diff
@@ -53,7 +53,7 @@ func GetResourceInfo(r *resource.Resource, repo, domain string) (resourcePackage
 		"setting":               "k8s.io",
 		"storage":               "k8s.io",
 	}
-	resourcePath := filepath.Join("api", r.Version, fmt.Sprintf("%s_types.go", strings.ToLower(r.Kind)))
+	resourcePath := filepath.Join("api", r.Group, r.Version, fmt.Sprintf("%s_types.go", strings.ToLower(r.Kind)))
```

Comment:
Should it be `api` or `apis` when it's multigroup?

Reply:
It is generated in the apis dir. We can check the updates in the testdata dir to ensure that. 
However, you make a good point here. I will check why here is api and not apis.

Reply:
By following the common practices/name conventions the nomenclature of the dirs should be always single if I am not wrong. However, any way I do not think we should change the name of the current dir in this PR.

Reply:
Make sense. Thanks.